src/Capgemini.PowerApps.PackageDeployerTemplate/Config/MailboxConfig.cs

```diff
@@ -0,0 +1,84 @@
+﻿namespace Capgemini.PowerApps.PackageDeployerTemplate.Config
+{
+    using System.Xml.Serialization;
+
+    /// <summary>
+    /// Mailbox status.
+    /// </summary>
+    public enum MailboxStatus
```

Comment:
I think `enum`s need to be split out into their own .cs files. Should be an analyzer warning.